### DISCUSSION TITLE
Just a list of some ideas/improvements

### DISCUSSION THREAD

**Discussion by o777l** [winstxnhdw/lc-hax#203] [Ideas]
Here are a few things I've been thinking of the past week, all may not be possible I know, but anyway. Fun potential.

• Ability to spawn store items into inventory or ground

• Spawn enemy of choice/on a player (seem like this is similar to already existing mask option, but it just tells me enemy hasn't been spawned yet?)

• Spawn mine on player

• Ragdoll a player with or without kill (last one may not work)

• M3 to trigger tentacle at company building

• M3 to duplicate items on ground

• Infinite carry/more inventory slots

• Fake thumper roar, enemy sounds

• Fake ghost girl apparition

• Fake delivery jingle

• Fake zero days quota message

• Attachables on self/player

• Spawn particles (fart lizard purple smoke, steam or emmissives)

**Comment by GigglingMinister** [comment]
m3 already trigger company tentacles
this is mark as jeb's attack
